### PR TITLE
Prep for 5.6

### PR DESCRIPTION
**Pull request by yadij**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,11 @@
+Changes in squid-5.6 (01 Jun 2022):
+
+	- Bug 5208: Part 1: Restart kids killed by SIGKILL
+	- Fix SQUID-MIB smilint errors
+	- negotiate_kerberos_auth: Initialise default_keytab
+	- Improve handling of Gopher responses
+	- ... and some semi-automated code polish
+
```
Comment (rousskov) [inline ChangeLog:8]: I would explicitly mention the added support for Gopher type "w" in the ChangeLog.

**Review by rousskov — COMMENTED** [review]
No objections from me. I do not insist on any changes.

File: ChangeLog (hunk 1 above)
Comment (yadij) [reply ChangeLog:8]: That commit is part of the generic "Improve handling of Gopher responses" line entry.

**Comment by yadij** [comment]
Delayed by review/commit process again, release date needs to be updated before it can be re-cleared for merge.

**Comment by rousskov** [comment]
> Delayed by review/commit process again

The PR was not delayed by reviews, and I would not blame the "process" for any significant delays here either, but if you want to speed up the process for this or any similarly non-controversial PR, then just explicitly ask core developers for a vote (addressing their concerns, if any, would be nice but is often not required). It is a really simple and fairly fast procedure that accommodates most use cases nicely. It is also possible to merge a PR without any cooperation; that naturally takes longer, but can be scheduled in advance.

Alternatively, we can adjust the change log maintenance process so that v5 change log updates are recorded in v5 without going through master/v6 first (and the log is then synced when, say, v6 is about to be branched). That would allow you, as a v5 maintainer, to commit change.log updates fast and without any cooperation from others. As a v5 maintainer, you know what works best for you; I am sure others would support any related process improvements!

**Comment by yadij** [comment]
> > Delayed by review/commit process again
> 
> The PR was not delayed by reviews, and I would not blame the "process" for any significant delays here either,

The record is clear. No reviewer approval for simple text update caused this to have to sit out the 10-day slow merge timeout.

**Comment by rousskov** [comment]
> > > Delayed by review/commit process again

> > The PR was not delayed by reviews, and I would not blame the "process" for any significant delays here either,
 
> The record is clear. No reviewer approval for simple text update caused this to have to sit out the 10-day slow merge timeout.

The record is indeed fairly clear. The true cause of slow merge despite a prompt non-blocking review is author inaction rather than the "process": Using the same process, one or more core developers would have fast-tracked this PR long time ago if the author explicitly asked them to do so to speed up merging (or if the author accepted the reviewer suggestion).

**Pull request closed without merging**